src/main/java/de/embl/cba/mobie/n5/zarr/N5S3ZarrReader.java

```diff
@@ -58,10 +58,15 @@
 	private static final String zgroupFile = N5ZarrReader.zgroupFile;
 
 	final protected boolean mapN5DatasetAttributes;
-	final protected String dimensionSeparator;
+
+	protected String dimensionSeparator;
 	private final String serviceEndpoint;
 
-	public N5S3ZarrReader( AmazonS3 s3, String serviceEndpoint, String bucketName, String containerPath ) throws IOException
+	public void setDimensionSeparator(String dimensionSeparator) {
+		this.dimensionSeparator = dimensionSeparator;
+	}
+
+	public N5S3ZarrReader(AmazonS3 s3, String serviceEndpoint, String bucketName, String containerPath ) throws IOException
```

Comment:
I think we should add another constructor which has the dimension separator as argument, similar to this https://github.com/mobie/mobie-viewer-fiji/blob/9b1693c437df77b948ed4ed2e588bafc3d15e3e0/src/main/java/de/embl/cba/mobie/n5/zarr/N5ZarrReader.java#L92.